### PR TITLE
Feature/ui-tests

### PR DESCRIPTION
**Pull request by niktrix**

### CODE REVIEW

File: apps/ethvm/src/components/blocks/largeBlocks/latestBlocks.vue [hunk 1]
```diff
@@ -149,6 +149,7 @@ export default Vue.extend({
     this.$eventHub.$on(sEvents.newBlock, _block => {
       if (Visibility.state() === 'visible') {
         this.blocks = this.$store.getters.getBlocks
+
```
Comment (aldoborrero) [inline apps/ethvm/src/components/blocks/largeBlocks/latestBlocks.vue]: Remove space

File: apps/ethvm/src/components/blocks/largeBlocks/blockDetail.vue [hunk 2]
```diff
@@ -216,11 +216,10 @@ export default Vue.extend({
   mounted() {
     this.setItems()
     this.setMore()
-    console.log(this.uncles)
   },
   computed: {
     isUncle() {
-      return this.block.getIsUncle()
+      return false
```
Comment (aldoborrero) [inline apps/ethvm/src/components/blocks/largeBlocks/blockDetail.vue]: Why is this removed?

Comment (niktrix) [reply apps/ethvm/src/components/blocks/largeBlocks/blockDetail.vue]: previously in getBlock was giving both Block and uncle, now there is a separate call to get uncle, so all block which are here are always Block not uncle
